storage-bigtable/src/bigtable.rs

```diff
@@ -680,17 +680,17 @@ impl<F: FnMut(Request<()>) -> InterceptedRequestResult> BigTable<F> {
         deserialize_protobuf_or_bincode_cell_data(&row_data, table, key)
     }
 
-    pub async fn get_protobuf_or_bincode_cells<B, P>(
+    pub async fn get_protobuf_or_bincode_cells<'a, B, P>(
         &mut self,
-        table: &str,
-        row_keys: &[RowKey],
-    ) -> Result<Vec<(RowKey, CellData<B, P>)>>
+        table: &'a str,
+        row_keys: impl Iterator<Item = String>,
```

Comment:
@mvines would `IntoIterator<RowKey>` make more sense here?  We could carry that through to `get_multi_row_data`

Reply:
Yeah that sounds nice.  Keep the API as `RowKey`-based and push any string conversions up to the caller :)

Reply:
Cool, maybe we just take this as is and I'll follow up with improvements

Reply:
addressed in 075ebcdca9239da0197742617f9d368657e0998e